Dockerfile

```diff
@@ -7,7 +7,6 @@ ARG H2ORESTART_CHECKSUM=5db816a1e57b510456633f55e693cb5ef3675ef8b35df4f31c90ab9d
 RUN apk --no-cache -U upgrade && \
     apk --no-cache add \
     ghostscript \
-    graphicsmagick \
```

Comment:
Do we remove only a single package? For instance, are `poppler` libraries still required? Are CJK fonts? BTW, it would be interesting measuring the impact of PyMuPDF in the resulting container image.